src/collections/Table/TableRow.js

```diff
@@ -76,6 +93,20 @@ TableRow.propTypes = {
   /** A row may call attention to an error or a negative value. */
   error: PropTypes.bool,
 
+  /** An element type to render as (string or function). */
+  itemAs: customPropTypes.as,
```

Comment:
Not sure of/if there is a solution yet, but I'd love to remove this prop somehow.  Seems an indicator of a larger pattern problem. We had this on the Label as well, but were able to remove it with sub components.

One idea would be that the implementing component would pass the `as` prop / key in each item in the array so it is explicit.  If that is too cumbersome, perhaps we need `TableHeaderRow` or something, similar to the two Table cells.